lib/views/words_view.dart

```diff
@@ -26,7 +24,7 @@ class WordsView extends StatelessWidget {
                 maxCrossAxisExtent: 125.0,
                 mainAxisSpacing: 10.0,
                 crossAxisSpacing: 10.0,
-                childAspectRatio: 113/166,
+                childAspectRatio: 113/160,
```

Comment:
not an error, just curious why it's "113/166", not a specific number

Reply:
Because the component designed by Henry originally have height 152 and width 113. 
But we need to increase the height a little bit to ensure the content in container won't overflow.

Reply:
ok👍